app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.kt

```diff
@@ -127,6 +127,11 @@ class ConversationItem(
         } else {
             holder.binding.favoriteConversationImageView.visibility = View.GONE
         }
+        if (ConversationEnums.ConversationType.ROOM_PUBLIC_CALL == model.type) {
+            holder.binding.publicCallBadge.visibility = View.VISIBLE
+        } else {
+            holder.binding.publicCallBadge.visibility = View.GONE
+        }
         if (ConversationEnums.ConversationType.ROOM_SYSTEM !== model.type) {
```

Comment:
Looking at my change in the lines above this seems no completely correct as in a public call can't have a user status (I guess) @mahibi @nickvergessen

Reply:
👍